components/scream/src/share/io/scream_output_manager.cpp

```diff
@@ -213,7 +213,7 @@ void OutputManager::run(const util::TimeStamp& timestamp)
       register_dimension(filename,"time","time",0);
 
       // Register time as a variable.
-      register_variable(filename,"time","time",1,{"time"},  PIO_REAL,"time");
+      register_variable(filename,"time","time","sec",1,{"time"},  PIO_REAL,"time");
```

Comment:
EKAT uses "s" for seconds, which is the standard symbol. All units printed by ekat should have 's'. Do climate app prefer 'sec'?

Reply:
Good point. I've fixed this to use "s" instead for consistency.